django/db/backends/base/schema.py

```diff
@@ -201,6 +202,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = datetime.now()
```

Comment:
Agreed but in the case of `DateTimeField` we should use `timezone.now()` instead just like [`DateTimeField.pre_save` does](https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L1449-L1452).